doc/codeExecution.md

```diff
@@ -0,0 +1,77 @@
+# Code Execution
```

Comment:
This is such a clear and well-written document!

Reply:
Thank you!!